bin/dotd

```diff
@@ -22,7 +22,8 @@ class String
   end
 end
 
-LOG_FILE = '/var/log/code-dot-org-dotd.log'
+BIN_DIR = File.expand_path(File.dirname(__FILE__))
+LOG_FILE = "#{BIN_DIR}/log/dotd.log"
```

Comment:
Hmm, unfortunately probably not; I had forgotten that was an intended use case. We could store the logfile alongside the script, which would put it in the `cdo/bin` directory for typical use case, but I'm not positive I like that.

Alternatively, we could go back to just dropping it in `/tmp/`